lib/miq_automation_engine/models/miq_ae_datastore.rb

```diff
@@ -191,6 +191,8 @@ def self.seed
         _log.info "Seeding... Complete"
       end
     end
+    _log.info "Reseting domain priorities at startup..."
+    MiqAeDomain.reset_priorities
```

Comment:
Should `reset_priorities` also be called after `reset_to_defaults` since that would be called after an upgrade which could introduce new domains?